### DISCUSSION TITLE
chat with local docs

### DISCUSSION THREAD

**Discussion by kalle07** [LostRuins/koboldcpp#882] [Q&A]
so you have decided to add image-generation, ok so that can be a hart part in complexity (like A1111 or similar) ^^

do you have the intention to add a chat with local docs, where i have a library like GPT4all, maybe i can choose also the model for embedding ... ??? i mean that is LLM suppose to do   ;)

**Comment by LostRuins** [comment]
Not at the moment - Kobold Lite is primarily a web UI, so it does not have access to your local documents.

**Comment by Amin456789** [comment]
thats a greaat idea, to go through a adventur from a long pdf with great tale or play a game with it, hope for support it
